Resources/doc/annotations/converters.rst

```diff
@@ -296,7 +296,11 @@ on the request attributes, it should set an attribute named
 ``$configuration->getName()``, which stores an object of class
 ``$configuration->getClass()``.
 
-To register your converter service, you must add a tag to your service:
+If you're using service `auto-registration and autoconfiguration`_,
+you're done! Your converter will automatically be used.
+If not, you must add a tag to your service:
+
+.. _auto-registration and autoconfiguration: http://symfony.com/doc/current/service_container/3.3-di-changes.html
```

Comment:
One minor thing, this (by our standard) needs to live all the way at the end of the file :)

Reply:
done thanks !